qiskit_metal/qlibrary/core/design_check.py

```diff
@@ -0,0 +1,83 @@
+### BASIC PROGRAM THAT LOOKS FOR OVERLAP BETWEEN QCOMPONENTS
+from numpy import size
+import geopandas as gpd
+import pandas as pd
+
+
+class QDesignCheck():
+    """QDesign_Check contains various design checks.
+    """
+
+    def __init__(self, design: 'QDesign'):
+        self.design = design
+
+    def update_design(self, design: 'QDesign'):
+        self.design = design
+
+    def overlap_tester(self):
+        design = self
+        """This particular function tests for overlap between qcomponents
```

Comment:
Doesn't this check also catch overlap between, say, two transmons?

Reply:
Indeed! I changed the description to include all collisions types: qubit/qubit, qubit/CPW, CPW/CPW.